cluster/gce/trusty/configure-helper.sh

```diff
@@ -147,9 +147,7 @@ assemble_kubelet_flags() {
   if [ -n "${KUBELET_PORT:-}" ]; then
     KUBELET_CMD_FLAGS="${KUBELET_CMD_FLAGS} --port=${KUBELET_PORT}"
   fi
-  if [ -n "${KUBELET_TEST_ARGS:-}" ]; then
-    KUBELET_CMD_FLAGS="${KUBELET_CMD_FLAGS} ${KUBELET_TEST_ARGS}"
-  fi
+  KUBELET_CMD_FLAGS="${KUBELET_CMD_FLAGS} ${KUBELET_TEST_ARGS:-}"
```

Comment:
Does it matter what order the flags are put on the command line? If not, then why not make link 143 above be `KUBELET_CMD_FLAGS="--v=2 ${KUBELET_TEST_ARGS:-}"` so that you can remove this block entirely?